apps/api_web/test/api_web/controllers/trip_controller_test.exs

```diff
@@ -296,7 +296,7 @@ defmodule ApiWeb.TripControllerTest do
                "id" => "trip",
                "relationships" => %{
                  "predictions" => %{
-                   "data" => [%{"id" => "prediction-trip-stop-", "type" => "prediction"}]
+                   "data" => [%{"id" => "prediction-trip-stop--", "type" => "prediction"}]
```

Comment:
nit: Would it make sense to add stop_sequence and route_id values to the Prediction struct created on line 275, so that they can show up in this string?

Reply:
I tried updating this test this morning. Various tests that rely on the State servers for trip (in the same file like line 59 or line 238, and in other files too, like api_view_helpers_test.exs line 37) started failing intermittently. This indicates an issue that's a test only, where the data leaks between tests (as State server data is expected to be shared between controllers in prod).